src/components/Main/TaskList/TaskForm/TaskForm.tsx

```diff
@@ -7,105 +7,85 @@ import CaretDown from "../../../../assets/img/caret-down.png";
-                                <button>
-                                    <img src={CaretUp} alt="" />
-                                </button>
-                                <button>
-                                    <img src={CaretDown} alt="" />
-                                </button>
-                            </div>
-                        </div>
-                    </div>
-                    <div className={classes["task-form-footer"]}>
-                        <div>
-                            <button className={classes["delete-button"]}>
-                                Delete
-                            </button>
-                        </div>
-                        <div>
-                            <button
-                                className={classes["cancel-button"]}
-                                onClick={() => this.handleCancelAction()}
-                            >
-                                Cancel
-                            </button>
-                            <button className={classes["save-button"]}>
-                                Save
-                            </button>
-                        </div>
-                    </div>
-                </div>
+  }, [openModal]);
+  return (
+    <div>
+      <div
+        className={`${classes["task-form-action"]} ${
+          !openModal ? "" : classes["display-none"]
+        }`}
+        onClick={() => handleOpenModal()}
+      >
+        <img src={AddTask} alt="add-task" />
+        <div className={classes["action"]}>Add Task</div>
+      </div>
+      <div
+        className={`${classes["task-form"]} ${
+          openModal ? "" : classes["display-none"]
+        }`}
+        ref={modalRef}
+      >
+        <div className={classes["task-form-body"]}>
+          <div className={classes["task-title-input"]}>
+            <input
+              className={classes["task-title"]}
+              type="text"
+              placeholder="Whare are you working on?"
+            />
+          </div>
+          <div className={classes["est-pomodoro"]}>
+            <div className={classes["est-pomodoro-title"]}>Est Pomodoros</div>
+            <div className={classes["est-pomodoro-action"]}>
+              <input
+                className={classes["est-pomodoro-input"]}
+                type="number"
+                placeholder="1"
+              />
+              <button>
+                <img src={CaretUp} alt="" />
+              </button>
+              <button>
+                <img src={CaretDown} alt="" />
+              </button>
             </div>
-        );
-    }
-}
+          </div>
+        </div>
+        <div className={classes["task-form-footer"]}>
+          <div>
+            <button className={classes["delete-button"]}>Delete</button>
+          </div>
+          <div>
+            <button
+              className={classes["cancel-button"]}
+              onClick={() => handleCancelAction()}
```

Comment:
Cuando tienes este caso particular donde no pasas props, puedes dejarlo de la siguiente forma.

```suggestion
              onClick={handleCancelAction}
```